MapboxNavigation/FeedbackItem.swift

```diff
@@ -7,10 +7,61 @@ extension UIImage {
     }
 }
 
+
+public extension FeedbackType {
+
+    // TODO: Localize these strings
```

Comment:
[To localize these strings](https://github.com/mapbox/mapbox-navigation-ios/blob/master/CONTRIBUTING.md), wrap each one in an `NSLocalizedString(_:bundle:value:comment:)`, as with the original strings that are being deleted, then run scripts/extract_localizable.sh to update all the Localizable.strings files.

Reply:
Will be handling in this in a follow up PR.

Reply:
Flagged this as action item as part of https://github.com/mapbox/mapbox-navigation-ios/pull/2429